flink-runtime/src/test/java/org/apache/flink/runtime/checkpoint/CheckpointCoordinatorTest.java

```diff
@@ -1028,7 +1029,7 @@ public void testTriggerAndDeclineCheckpointComplex() throws Exception {
                 checkpointCoordinator.getPendingCheckpoints().get(checkpoint2Id);
 
         assertNotNull(checkpoint1);
-        assertEquals(checkpoint1Id, checkpoint1.getCheckpointId());
+        assertEquals(checkpoint1Id, checkpoint1.getCheckpointID());
```

Comment:
getCheckpointId() is a deprecated method. Code snippet below - 
```
/** @deprecated use {@link #getCheckpointID()} */
   @Deprecated
   public long getCheckpointId() {
       return getCheckpointID();
   }
```
So, I have updated it to use the getCheckpointID() method.